src/frontend/sharedComponents/CustomHeaderLeftClose.tsx

```diff
@@ -61,17 +27,20 @@ interface SharedBackButtonProps {
 
 type CustomHeaderLeftCloseProps = {
   observationId?: string;
+  openDiscardModal: () => void;
```

Comment:
seeing this being added is making me notice the scope creep of this component, but i think addressing that is outside the scope of this PR. can create an issue to provide more details of what a refactor could entail